e2e/core/login/login-sso/login-sso.e2e.ts

```diff
@@ -19,6 +19,7 @@ import { LoginSSOPage, SettingsPage, LoginPage } from '@alfresco/adf-testing';
 import { browser } from 'protractor';
 import { NavigationBarPage } from '../../../pages/adf/navigationBarPage';
 
+/* tslint:disable */
```

Comment:
seems to be disabling the whole tslint here

Reply:
yes the test is excluded I am going to create a task to fix it but we already are working on it on another branch with @adinapitul

Reply:
I can anyway try do to another fix on those too coping form there